packages/rashell/rashell.0.1.0/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+version: "0.1.0"
+maintainer: "michipili@gmail.com"
+authors: "Michael Grünewald"
+license: "LGPL"
```

Comment:
What is this linking exception? My intent was to use the same licence as Lwt because I had to borrow a few functions. Otherwise I stick to BSD-style licences, probably to avoid this kind of subtleties.